src/ktn_dodger.erl

```diff
@@ -331,7 +334,7 @@ quick_parse_form(Dev, L0) ->
 quick_parse_form(Dev, L0, Options) ->
     parse_form(Dev, L0, fun quick_parser/2, Options).
 
--type pre_fixer() :: fun((erl_scan:tokens()) -> no_fix | {retry, erl_scan:tokens()}).
+-type pre_fixer() :: fun(([erl_scan:token()]) -> no_fix | {retry, [erl_scan:token()]}).
```

Comment:
I think this is an Ok-compromise: 23-26 all support `:token/0` whereas `:tokens/0` is 24+.

Reply:
Another reason to drop support for OTP23.

Reply:
9e7953f